emukit/bayesian_optimization/acquisitions/probability_of_improvement.py

```diff
@@ -67,3 +67,56 @@ def evaluate_with_gradients(self, x: np.ndarray) -> Tuple:
     @property
     def has_gradients(self):
         return isinstance(self.model, IDifferentiable)
+    
+    
+    
+class ProbabilityOfFeasibility(Acquisition):
+
+    def __init__(self, model: Union[IModel, IDifferentiable], jitter: np.float64 = np.float64(0)) -> None:
+        """
+        This acquisition computes for a given input point the probability of satisfying the constraint
+        C<0. For more information see:
+
+        Michael A. Gelbart, Jasper Snoek, and Ryan P. Adams,
+        Bayesian Optimization with Unknown Constraints,
+        https://arxiv.org/pdf/1403.5607.pdf
+
+        :param model: The underlying model that provides the predictive mean and variance for the given test points
+        :param jitter: Jitter to balance exploration / exploitation
+        """
+        self.model = model
+        self.jitter = jitter
+
+    def evaluate(self, x: np.ndarray) -> np.ndarray:
+        """
+        Computes the probability of improving over the current best
+
+        :param x: points where the acquisition is evaluated.
+        """
+        mean, variance = self.model.predict(x)
+        standard_deviation = np.sqrt(variance)
+        _, cdf, _ = get_quantiles(self.jitter, 0, mean, standard_deviation)
```

Comment:
We have removed the get_quantiles import from this file so we don't have to rely on GPyOpt so this line won't run! Could you look at the code for Probability of improvement acquisition and try to factor out the relevant bits so you can reuse it here?